doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -112,6 +112,7 @@ If you are using a MySQL version 8 image provided by Red Hat on {OPENSHIFT}, to
 If you created users on the MySQL version 8 server before enabling the `mysql_native_password` plugin, you must update the `mysql-user` table after you enable the plugin.
 endif::PAM[]
 . Optionally, depending on your needs, set environment variables. To set an environment variable, click *Add new Environment variable*, then enter the name and value for the variable in the *Name* and *Value* fields.
+** If you want to configure an immutable KIE server that pulls services from the configured Maven repository, set the `KIE_SERVER_CONTAINER_DEPLOYMENT` environment variable. The variable must contain the identifying information of the services (KJAR files) that the deployment must pull from the Maven repository. The format is `<containerId>=<groupId>:<artifactId>:<version>` or, if you want to specify an alias name for the container, `<containerId>(<aliasId>)=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, as illustrated in the following example: `containerId=groupId:artifactId:version|c2(alias2)=g2:a2:v2`.
```

Comment:
add a link to the section where the maven repo configuration is explained.

Reply:
Sadly, a link is not exactly possible within the same section, but I will add a note that this has to be configured.

Reply:
The note was now added

Reply:
thanks.